lib/ansible/modules/monitoring/zabbix/zabbix_host_facts.py

```diff
@@ -54,6 +54,13 @@
             - Remove duplicate host from host result
         type: bool
         default: yes
+    host_inventory:
+        description:
+            - Specify host inventory name.
```

Comment:
Current description is not very clear to me, I assume it was meant to be something like this: 
```yaml
- List of host inventory keys to display in result
- Whole host inventory is retrieved if keys are not specified
```

Reply:
Thank you.
I will modify the description.